www/src/examples/ProgressBarAnimated.js

```diff
@@ -1 +1 @@
-<ProgressBar active now={45} />;
+<ProgressBar animated now={45} />;
```

Comment:
lets also move the examples to `examples/ProgressBar` like we've started to do with the other components 👍